lib/src/main/java/com/auth0/jwt/creators/AccessJwtCreator.java

```
@@ -191,7 +185,11 @@ public AccessJwtCreator withNonStandardClaim(String name, Double value) throws I
      * @throws IllegalArgumentException if the name is null.
      */
     public AccessJwtCreator withNonStandardClaim(String name, Date value) throws IllegalArgumentException {
-        jwt.withNonStandardClaim(name, value);
+        if(name.equalsIgnoreCase(Claims.ISSUED_AT) || name.equalsIgnoreCase("issuedAt") || name.equalsIgnoreCase("issued_at")) {
```

Just check the Claims constant value.

---

again, whats stopping someone from supplying "issuedAt"?  should we throw an exception?

---

they can but that's not standardized, so we don't check the type of those. Stick to the JWT RFC

---

so it just falls into the jwt.withNonStandardClaim(name, value); block?